I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/story/hallowed winds/hw_day_1.rpy", line 126, in script
    show image hw_splash with moveintop
  File "game/story/hallowed winds/hw_day_1.rpy", line 126, in <module>
    show image hw_splash with moveintop
NameError: name 'hw_splash' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/story/hallowed winds/hw_day_1.rpy", line 126, in script
    show image hw_splash with moveintop
  File "C:\Program Files\renpy-8.1.1-sdk\renpy\ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Program Files\renpy-8.1.1-sdk\renpy\ast.py", line 1384, in show_imspec
    expression = renpy.python.py_eval(expression)
  File "C:\Program Files\renpy-8.1.1-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Program Files\renpy-8.1.1-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/story/hallowed winds/hw_day_1.rpy", line 126, in <module>
    show image hw_splash with moveintop
NameError: name 'hw_splash' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Production 1.0
Tue Mar 12 14:17:23 2024
